javaslang/src/main/java/javaslang/collection/Array.java

```diff
@@ -7,6 +7,7 @@
 
 import javaslang.*;
 import javaslang.collection.ArrayModule.Combinations;
+import javaslang.collection.JavaConverters.SeqAsJavaList;
```

Comment:
Just as a general note (nothing to do with this line): a tool like reviewable.io might reviews like this one a lot easier.